test/basic-querying.test.js

```diff
@@ -467,7 +467,7 @@ describe('basic-querying', function() {
       return User.find({where: {seq: {inq: [0, 1, 100]}}})
         .then(function(result) {
           var seqsFound = result.map(function(r) { return r.seq; });
-          should(seqsFound).eql([0, 1]);
+          should(seqsFound.sort()).containDeep([0, 1]);
```

Comment:
I think `containDeep` allows the array to have extra elements, e.g. `100` which should not be included. Is there any particular reason why `.eql` cannot be used now that the array items are sorted?